rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -295,6 +290,27 @@ namespace rocal{
         return py::cast<py::none>(Py_None);
     }
 
+    '''
+        std::unordered_map<int, std::string> rocALToPybindLayout = {
+        {0, "NHWC"},
+        {1, "NCHW"},
+        {2, "NFHWC"},
+        {3, "NFCHW"},
+    };
+
+    std::unordered_map<int, int> rocALToPybindImageColor = {
+        {0, 3},
+        {1, 3},
+        {2, 1},
+        {3, 3},
+    };
+
+    std::unordered_map<int, std::string> rocALToPybindOutputDtype = {
```

Comment:
I think these named need to be rocalToPybindOutputDtype not rocAL, please check with @shobana-mcw and chane

Reply:
changed rocAL to rocal